packages/protocol/contracts/tko/BridgedTaikoToken.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import "../tokenvault/IBridgedERC20.sol";
+import "./TaikoTokenBase.sol";
+
+/// @title BridgedTaikoToken
+/// @notice The TaikoToken on L2 to support checkpoints and voting. For testnets, we do not need to
+/// use this contract.
```

Comment:
Wondering why we need this (special) for TaikoToken ? The idea is now, to remove Votes from bridged tokens ? (Except bridged TKO)

Reply:
yes